client/src/Movies/MovieList.js

```diff
@@ -33,14 +34,14 @@ export default class MovieList extends Component {
 
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
-  return (
-    <div className="movie-card">
-      <h2>{title}</h2>
-      <div className="movie-director">
-        Director: <em>{director}</em>
-      </div>
-      <div className="movie-metascore">
-        Metascore: <strong>{metascore}</strong>
+  return (<Link to={`/movie.id}`}>
```

Comment:
Great job using the backticks here! The next step to get this working will be to use interpolation to add JavaScript into this string. Remember that we can access variables inside of string template literals by using `${}`.